misc/docerina/src/test/java/org/ballerinalang/docgen/tests/GenDocsForBalaTest.java

```diff
@@ -106,7 +106,7 @@ public void generatingDocsForBalaWithAnnotationTest() throws IOException {
     }
 
     @Test
-    public void generatingDocsForBalaWithAnnotationTest2() throws IOException {
+    public void generatingDocsForBalaWithAnnotationTest2() throws Exception {
```

Comment:
Shall we add a test case for a package containing compilation errors?

Reply:
Added in [8534b2f](https://github.com/ballerina-platform/ballerina-lang/pull/38661/commits/8534b2f6e273dca573ee2d3345dffdd6e29e371e)